tests/behat/fixtures/blackbox/drag_and_drop.html

```
@@ -0,0 +1,34 @@
+<html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)